bin/BUILD

```diff
@@ -3,10 +3,10 @@ load("@build_bazel_rules_nodejs//:defs.bzl", "nodejs_binary")
 
 nodejs_binary(
     name = "protoc-gen-ts",
+    node_modules = "@deps//:node_modules",
     data = [
-        "@//:node_modules",
-        "@//src",
+      "//src:src",
     ],
-    entry_point = "ts_protoc_gen/src/ts_index",
```

Comment:
Whoops, good catch. I'm surprised this didn't fail CI, is there a way we can automatically test for errors like this?

Reply:
I'm actually surprised it wasn't caught either, not sure why, and I don't really have the resources to tell you why.
The previous change `bazel build //...:all` is a good way to ensure bazel builds - it would also end up running tests you eventually create with bazel.